src/GTP.cpp

```diff
@@ -345,12 +349,18 @@ bool GTP::execute(GameState & game, std::string xinput) {
             }
         }
         return true;
-    } else if (command.find("genmove") == 0) {
+    } else if (command.find("genmove") == 0 || command.find("lz-genmove_analyze") == 0 ) {
+        bool analysis_output = command.find("lz-genmove_analyze") == 0;
```

Comment:
auto 

(I know some of the existing code doesn't use it, but avoid adding new)